modules/hosted-control-planes-monitoring-dashboard.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-monitoring-dashboard_{context}"]
+= Creating monitoring dashboards for hosted clusters
+
+The HyperShift Operator can create or delete monitoring dashboards in the management cluster for each hosted cluster that it manages.
+
+[#hosted-control-planes-enable-dashboard]
+== Enabling monitoring dashboards
+
+To enable monitoring dashboards in a hosted cluster, complete the following steps:
+
+. Create the `hypershift-operator-install-flags` config map in the `local-cluster` namespace, being sure to specify the `--monitoring-dashboards` flag in the `data.installFlagsToAdd` section. For example:
+
++
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: hypershift-operator-install-flags
+  namespace: local-cluster
+data:
+  installFlagsToAdd: "--monitoring-dashboards"
+  installFlagsToRemove: ""
+----
+
++
+Within a couple of minutes, the HyperShift Operator deployment in the `hypershift` namespace is updated to include the following environment variable:
+
++
+----
+    - name: MONITORING_DASHBOARDS
+      value: "1"
+----
+
```

Comment:
When monitoring dashboards are enabled, the HyperShift operator creates a configmap named cp-[NAMESPACE]-[NAME] in the openshift-config-managed namespace (where NAMESPACE is the namespace of the HostedCluster and NAME is the name of the HostedCluster) for each HostedCluster that the operator manages. This results in a new dashboard getting added under Observe -> Dashboards in the administrative console of the management cluster. When a HostedCluster is deleted, its corresponding dashboard is also deleted.When monitoring dashboards are enabled, the HyperShift operator creates a configmap named cp-[NAMESPACE]-[NAME] in the openshift-config-managed namespace (where NAMESPACE is the namespace of the HostedCluster and NAME is the name of the HostedCluster) for each HostedCluster that the operator manages. This results in a new dashboard getting added under Observe -> Dashboards in the administrative console of the management cluster. When a HostedCluster is deleted, its corresponding dashboard is also deleted.